html/templates/schematics.html

```diff
@@ -305,6 +305,11 @@ <h1>Crafting Schematics
 {% endif %}
 
 {% if (schematicID != '' and schematicID != 'index' and schematicID != 'home') %}
+    {% if schematic.schematicID and schematic.galaxy != 0 and schematic.galaxy|string != selectedGalaxy %}
```

Comment:
Needs a conditional check for NGE servers as well. Maybe move this conditional logic back to `schematics.py` and pass the boolean down to the template instead.